tools/bqtag/tests/test_bqtag.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2019 Google LLC
```

Comment:
Copyright 2021

Reply:
Changed to:

Copyright 2021 Google LLC